Changes.md

```diff
@@ -1,6 +1,11 @@
 0.61.x.x (relative to 0.61.3.0)
 ========
 
+Features
+--------
+
+- GafferImage : Added new node for choosing a random value from a list of weighted choices.
```

Comment:
- FormatQuery : Added new node for querying the format of an image and outputting its displayWindow, pixelAspect, center and size.